MAUIsland/Features/Gallery/MAUI/RaditoButton/Pages/RadioButtonPage.xaml

```diff
@@ -7,10 +7,428 @@
-    </VerticalStackLayout>
+            HorizontalOptions="Start"
+            FontAttributes="None"
+            Padding="10"
+            FontSize="Default"/>
+            <Frame Style="{x:StaticResource DocumentContentFrameStyle}"
+               Padding="20,10,20,10">
+                <VerticalStackLayout Spacing="0">
+                    <Grid Margin="0,0,0,5">
+                        <Grid.Resources>
+                            <FontImageSource x:Key="ShowCode"
+                                             Color="{x:StaticResource Primary}"
+                                             FontFamily="{x:Static app:FontNames.FluentSystemIconsRegular}"
+                                             Glyph="{Static app:FluentUIIcon.Ic_fluent_code_16_regular}"
+                                             Size="25"/>
+                        </Grid.Resources>
+                        <Label Text="Basic radio button" Margin="0,0,0,5"
+                           FontSize= "Medium"
+                           HorizontalOptions="Start"
+                           VerticalOptions="Center"/>
+
+                        <Border HorizontalOptions="End"
+                            Stroke="White"
+                            StrokeShape="RoundRectangle 10,10,0,0">
+                            <Button x:Name="showCode1"
+                                    HeightRequest="20"
+                                    BackgroundColor="Transparent"
+                                    Text="Show Code"
+                                    ImageSource="{x:StaticResource ShowCode}"
+                                    Clicked="Button_Clicked">
+                            </Button>
+                        </Border>
+
+                        <Grid HeightRequest="1" 
+                             VerticalOptions="End"
+                             Background="White"/>
+                    </Grid>
+
+                    <StackLayout x:Name="example1">
+                        <RadioButton Content="Cat"
+                             Margin="0,0,0,-10"
+                             IsChecked="True"/>
+                        <RadioButton Content="Dog" 
+                             Margin="0,0,0,-10"/>
+                        <RadioButton Content="Fish" 
+                             Margin="0,0,0,-10"/>
+                        <RadioButton Content="Bird" 
+                             Margin="0,0,0,-10"/>
+                    </StackLayout>
+
+                    <Grid x:Name="code1" 
+                          IsVisible="False">
+                        <Editor Text="{x:Binding StandardRadioButtonXamlCode}"
+                                AutoSize="TextChanges"
+                                IsReadOnly="True"/>
+                    </Grid>
+                    <app:SourceCodeExpander XamlCode="{x:Binding StandardRadioButtonXamlCode}"/>
+                </VerticalStackLayout>
+            </Frame>
+
+            <ListView  Margin="10"
```

Comment:
Please change this to a `CollectionView`, just the control name all the properties below won't require to change anything